scalableminds/webknossos
# Lazy load onnx and canvas2html module

I also extracted the html2canvas module since it caught my attention in the bundle analyzer and was simple to extract.

### URL of deployed dev instance (used for testing):
- https://lazyonnx.webknossos.xyz

### Steps to test:
- open an annotation and the network tab
- select the quick select tool (onnx module and ml model should be loaded)
- use the tool
- hit Q to create a screenshot (html2canvas should be loaded)
- screenshot should work

### Issues:
- fixes #7084 

------
(Please delete unneeded items, merge only when none are left open)
- [ ] ~Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)~ doesn't seem changelog-worthy to me
